common/sgx/workload/workload_crypto_utils.cpp

```diff
@@ -0,0 +1,81 @@
+/* Copyright 2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <string>
+
+#include "crypto.h"
+#include "error.h"
+#include "tcf_error.h"
+#include "types.h"
+#include "utils.h"
+#include "hex_string.h"
+#include "enclave_utils.h"
+#include "workload_crypto_utils.h"
+
+// Create symmetric encryption key and return hex encoded key string
```

Comment:
Please avoid global functions. Implement functions inside C++ class and namespace.
I would suggest to move the files crypto_utils.cpp/.h inside simple_wallet/workload. The main crypto functionalities are already in  tcf::crypto: module.

Reply:
these are generic crypto util files can be utilized from different workloads. Hence, it can't be moved inside simple_wallet/workload module.

Reply:
Generic crypto functions should be kept in crypto module in tc/sgx/common/crypto.

Reply:
this is kind of wrapper file to make use of crypto functions. I wanted to seperate usage of crypto functions from simple wallet workload, hence i added this wrapper. Actual implementation of Encryption/Decryption can be part of tc/sgx/common/crypto.

Reply:
There is already crypto_utils.cpp in tc/sgx/common/crypto. You can add these functions in same file. Else we will have multiple related crypto functionalities in multiple places.  Also as of now, I see only simple wallet workload making use of these files. So I would suggest either move it to simple_wallet workload or move it to tc/sgx/common/crypto.

Reply:
Ok, i will add these functions to crypto_utils.cpp in tc/sgx/common/crypto